### PR TITLE
Update rubocop and rubocop-performance

### PR DESCRIPTION
**Pull request by dorotheelaugwitz**
* Update versions of `rubocop` and `rubocop-performance`
* Update `.rubocop.yml` to match current shipcloud style guides
* Correct styling issues automatically with `rubocop -a`

### CODE REVIEW

File: pactas_itero.gemspec [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
-    the Pactas Itero API, making it easy to post your data to, adn read your
-    data from your Pactas account.}
-  spec.homepage      = 'https://github.com/webionate/pactas_itero'
-  spec.license       = 'MIT'
+  spec.summary       = 'pactas_itero provides a client mapping for accessing
+    the Pactas Itero API.'
+  spec.description   = 'pactas_itero provides a client mapping for accessing
+    the Pactas Itero API, making it easy to post your data to, and read your
+    data from your Pactas account.'
+  spec.homepage      = "https://github.com/webionate/pactas_itero"
```
Comment (FranziGrieser) [inline pactas_itero.gemspec:16]: maybe we can create a follow-up story to update these outdated description?

**Review by FranziGrieser — CHANGES_REQUESTED** [review]
What do you think about the "missing magic comment"-HoundLints. Maybe we can add these although it's a Hound suggestion? I thought we have that rubocop-Lint as well 🤔 

Can you rebase the PR to fix the Changelog and Gemspec conflicts?

**Comment by dorotheelaugwitz** [comment]
> What do you think about the "missing magic comment"-HoundLints. Maybe we can add these although it's a Hound suggestion?

These can potentially break things so I would rather add them in a separate commit. I'll write a ticket for that.

File: pactas_itero.gemspec (hunk 1 above)
Comment (dorotheelaugwitz) [reply pactas_itero.gemspec:16]: Yes, good idea. Will do! :+1:

**Review by FranziGrieser — APPROVED** [review]
Thanks for the clean up 🥳 

RTM

**Pull request merged**